homeassistant/components/lirc.py

```diff
@@ -0,0 +1,78 @@
+"""
+LIRC interface to receive signals from a infrared remote control.
+
+This sensor will momentarily set state to various values as defined
+in the .lintrc file which can be interpreted in home-assistant to
+trigger various actions.
+
+Sending signals to other IR receivers can be accomplished with the
+shell_command component and the irsend command for now.
+"""
+# pylint: disable=import-error
+import threading
+import time
+import logging
+
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+
+DOMAIN = "lirc"
+REQUIREMENTS = ['python-lirc>=1.2.1']
+_LOGGER = logging.getLogger(__name__)
+ICON = 'mdi:remote'
+EVENT_BUTTON_PRESSED = 'button_pressed'
```

Comment:
Let's not call it button pressed but IR_COMMAND_RECEIVED ? Not all IR commands will come in because of a button press.